From c4b914a410583822234f214a46dfdc72e8582e37 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 11:36:51 -0400
Subject: [ethernet] i40e: Fix a message string

Message-id: <1409658042-22689-51-git-send-email-sassmann@redhat.com>
Patchwork-id: 90244
O-Subject: [RHEL7.1 PATCH 050/281] i40e: Fix a message string
Bugzilla: 1091126
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Anjali Singhai Jain <anjali.singhai@intel.com>

Change string from "Side Band" to "Sideband" for consistency.

Change-ID: I45f05466bb5c63b0f999d743312bcb61b5bd6518
Signed-off-by: Anjali Singhai Jain <anjali.singhai@intel.com>
Signed-off-by: Catherine Sullivan <catherine.sullivan@intel.com>
Tested-by: Kavindya Deegala <kavindya.s.deegala@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 0b67584f8047cf5abdad23a4ee1558c488684752)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_main.c b/drivers/net/ethernet/intel/i40e/i40e_main.c
index 4791bb3..44f88c8 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_main.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_main.c
@@ -6396,7 +6396,7 @@ static int i40e_sw_init(struct i40e_pf *pf)
    pf->flags |= I40E_FLAG_FD_SB_ENABLED;
   } else {
    dev_info(&pf->pdev->dev,
-     "Flow Director Side Band mode Disabled in MFP mode\n");
+     "Flow Director Sideband mode Disabled in MFP mode\n");
   }
   pf->fdir_pf_filter_count =
      pf->hw.func_caps.fd_filters_guaranteed;
-- 
1.7.1